### ISSUE TITLE
SC2054 - Use spaces, not commas, to separate array elements

### ISSUE THREAD

**Issue by dusanbrankov** [koalaman/shellcheck#2889]
#### For bugs

- [SC2054](https://www.shellcheck.net/wiki/SC2054)
- shellcheck version: 0.8.0 
- [x] The rule's wiki page does not already cover this
- [x] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### Here's a snippet that shows the problem:

```sh
#!/bin/bash

fzf_opts=(  # "Use spaces, not commas, to separate array elements."
    --multi
    --reverse
    --cycle
    --border=none
    --prompt='→ '
    --pointer=' '
    --marker='* '
    --color=dark,fg+:white,fg:white,bg+:-1,marker:red,info:grey
    --height=90%
    --min-height=10
    --padding=0,8%,13%,3%
    --preview='git diff $@ --color=always -- {-1}'
    --preview-window="${fzf_preview_pos:-right},border-sharp"
    --bind=ctrl-p:toggle-preview
    --bind=ctrl-a:select-all,ctrl-d:deselect-all
)
```

Why does Shellcheck complain? There are no commas, so what exactly is causing the output of this warning?

**Edit:** Ok, I need to double quote the strings that contain commas, but it should still be clear that I am not using commas to separate the array elements given in the example above.

**Comment by heftig** [comment]
Can this be reopened? I find this check unnecessarily strict as well.

**Comment by wileyhy** [comment]
Commas aren't used very often in bash's syntax, speaking one of the
relevant shells, but as a workaround, quoting any array value strings which
include commas quiets this SC2054 error on shellcheck.net.
